esmvalcore/_recipe.py

```diff
@@ -7,7 +7,9 @@
 from copy import deepcopy
 from pprint import pformat
 
+import isodate
```

Comment:
This package is not yet listed in the dependencies, could you add it please?

Reply:
I did not have to install it as a dependency. I think it must come with the standard Python.

Reply:
It is probably a dependency of one of our dependencies. However, because other packages might change their dependencies at any point without notifying us, it is important that we declare the complete list of our own dependencies. You can find the modules that are part of the Python standard library [here](https://docs.python.org/3/library/). The isodate package is an independently developed package that can be installed from PyPI, see [here](https://pypi.org/project/isodate/).